es-core/src/components/VideoComponent.cpp

```diff
@@ -221,6 +212,15 @@ void VideoComponent::applyTheme(const std::shared_ptr<ThemeData>& theme, const s
 	if (elem->has("showSnapshotDelay"))
 		mConfig.showSnapshotDelay = elem->get<bool>("showSnapshotDelay");
 
+	setRotation(0.0);
```

Comment:
I was convinced that this would be necessary, but I did a bunch of testing and it wasn't so I have removed it.